charts/tidb-drainer/values.yaml

```
@@ -28,7 +28,7 @@ storage: 10Gi
 # disbale detect causality
 disableDetect: false
 # if drainer donesn't have checkpoint, use initial commitTS to initial checkpoint
-initialCommitTs: 0
+initialCommitTs: "0"
```

```suggestion
initialCommitTs: "-1"
```

Do you means change it to "-1"? @DanielZhangQD

---

actuallly